machine.md

```diff
@@ -252,6 +252,33 @@ spec:
       persistentVolumeClaim:
         claimname: myclaim
 ```
+## Resources Requests and Limits
+
+An optional Resources requests can be specified by the users to
+allow the scheduler to make a better decision in finding the most suitable Node
+to place the Virtual Machine on.
+
+```
+    resources:
```

Comment:
Please extend the example with the `cpu:` field to highlight the difference beween the two